languages/pt_BR.json

```diff
@@ -63,7 +63,7 @@
   "TXT_CODE_daemonInfo.authSuccess": "Validação da chave do daemon remoto {{- v}} bem-sucedida",
   "TXT_CODE_daemonInfo.closed": "Daemon remoto {{- v}} desconectado",
   "TXT_CODE_daemonInfo.connect": "Conectado ao daemon remoto {{- v}}",
-  "TXT_CODE_daemonInfo.connectError": "Erro ao conectar ao daemon remoto: {{- v}}",
+  "TXT_CODE_daemonInfo.connectError": "Erro ao conectar ao daemon remoto {{- v}}: {{error}}",
```

Comment:
Does this place support `{{error}}` ? Are you sure you don't want `{{- error}}` ?